tasks/connectors/edgescan/edgescan.rb

```diff
@@ -69,7 +79,11 @@ def run(opts)
         edgescan_api.fetch_in_batches do |edgescan_assets, edgescan_definitions|
           edgescan_assets.each do |edgescan_asset|
             kenna_api.add_assets(edgescan_asset)
-            kenna_api.add_vulnerabilities(edgescan_asset.vulnerabilities)
+            if @options[:create_findings]
```

Comment:
Should the `--create-findings` flag disable the import of vulnerabilities, or should it simply enable the import of findings?

Reply:
As far as I can tell edgescan vulnerabilities should be imported as vulnerabilities OR findings.

From the feature request:
"Kenna are updating their data model and have asked us to add a toggle that controls whether to create vulnerabilitys or findings for apps ([the differences are outlined here](https://help.kennasecurity.com/hc/en-us/articles/360026413111-Kenna-Data-Importer-JSON-Connector-))"

Reply:
@annapoltieva: are vulnerabilities and findings mutually exclusive?

Reply:
Sorry to leave you hanging, but I still discuss that with the business

Reply:
The answer - We definitely want findings OR vulns. Findings is the new model, vulns is old model (for AppSec Only. for VM it will always be vulns).